pages/[username].js

```diff
@@ -1,146 +1,155 @@
-import Head from "next/head";
-import ReactMarkdown from "react-markdown";
-import { abbreviateNumber } from "js-abbreviation-number";
+import Head from 'next/head';
+import ReactMarkdown from 'react-markdown';
+import { abbreviateNumber } from 'js-abbreviation-number';
 
-import app from "../config/app.json";
-import SingleLayout from "../components/layouts/SingleLayout";
-import MultiLayout from "../components/layouts/MultiLayout";
-import singleUser from "../config/user.json";
-import UserLink from "../components/user/UserLink";
-import UserMilestone from "../components/user/UserMilestone";
-import FallbackImage from "../components/FallbackImage";
-import EventPreview from "../components/events/EventPreview";
-import UserSocial from "../components/user/UserSocials";
+import app from '../config/app.json';
+import SingleLayout from '../components/layouts/SingleLayout';
+import MultiLayout from '../components/layouts/MultiLayout';
+import singleUser from '../config/user.json';
+import Link from 'next/link';
+import UserLink from '../components/user/UserLink';
+import UserMilestone from '../components/user/UserMilestone';
+import FallbackImage from '../components/FallbackImage';
+import EventPreview from '../components/events/EventPreview';
+import UserSocial from '../components/user/UserSocials';
 
 export async function getServerSideProps(context) {
-  let data = {};
-  try {
-    const res = await fetch(
-      `${app.baseUrl}/api/users/${context.query.username}`
-    );
-    data = await res.json();
-  } catch (e) {
-    console.log("ERROR user not found ", e);
-  }
+    let data = {};
+    try {
+        const res = await fetch(
```

Comment:
I think the coding style of the project is 2 spaces not 4

It looks like there are more changes which makes it difficult to review

Reply:
I just changed some of my editor settings now, I will push changes soon

Reply:
should be fixed now